var/spack/repos/builtin/packages/postgresql/package.py

```diff
@@ -125,3 +125,22 @@ def setup_dependent_run_environment(self, env, dependent_spec):
             env.prepend_path('TCLLIBPATH', self.prefix.lib)
         if '+python' in spec:
             env.prepend_path('PYTHONPATH', self.prefix.lib)
+
+    @property
+    def libs(self):
+        stat_libs = ['libecpg_compat', 'libecpg', 'libpgcommon',
+                     'libpgcommon_shlib', 'libpgfeutils', 'libpgport',
+                     'libpgport_shlib', 'libpgtypes', 'libpq']
+        fl_stat = find_libraries(stat_libs, self.prefix, shared=False,
+                                 recursive=True)
+
+        dyn_libs = ['libecpg_compat', 'libecpg', 'libpgtypes', 'libpq',
+                    'libpqwalreceiver', 'plpgsql', 'pgoutput']
+        dyn_libs.append('plperl') if '+perl' in self.spec
+        dyn_libs.append('plpython') if '+python' in self.spec
+        dyn_libs.append('pltcl') if '+tcl' in self.spec
```

Comment:
This isn't valid Python. I think you want:
```suggestion
        if '+perl' in self.spec:
            dyn_libs.append('plperl')
        if '+python' in self.spec:
            dyn_libs.append('plpython')
        if '+tcl' in self.spec:
            dyn_libs.append('pltcl')
```

Reply:
oops.